airflow/bin/cli.py

```diff
@@ -90,7 +90,10 @@ def sigquit_handler(sig, frame):
 
 def setup_logging(filename):
     root = logging.getLogger()
-    handler = logging.FileHandler(filename)
+    handler_buffer_size = conf.getInt('core', 'logging_buffer_size')
```

Comment:
Shoot. Didn't realize this code path was only hit with `-D`. Ill 'rpobably revert this